pkg/controller/toolchainstatus/toolchainstatus_controller.go

```diff
@@ -400,6 +428,48 @@ func (r *ReconcileToolchainStatus) sendToolchainStatusUnreadyNotification(logger
 	return nil
 }
 
+func (r *ReconcileToolchainStatus) sendToolchainStatusRestoredNotification(logger logr.Logger,
+	toolchainStatus *toolchainv1alpha1.ToolchainStatus) error {
+
+	if !isValidEmailAddress(r.config.GetAdminEmail()) {
+		return errs.New(fmt.Sprintf("cannot create notification due to configuration error - admin.email [%s] is invalid or not set",
+			r.config.GetAdminEmail()))
+	}
+	toolchainStatus = toolchainStatus.DeepCopy()
+	toolchainStatus.ManagedFields = nil // we don't need these managed fields in the notification
+	statusYaml, err := yaml.Marshal(toolchainStatus)
+	if err != nil {
+		return err
+	}
+
+	tsValue := time.Now().Format("20060102150405")
+
+	notification := &toolchainv1alpha1.Notification{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      fmt.Sprintf("toolchainstatus-ready-%s", tsValue),
+			Namespace: toolchainStatus.Namespace,
+		},
+		Spec: toolchainv1alpha1.NotificationSpec{
+			Recipient: r.config.GetAdminEmail(),
+			Subject:   adminReadyNotificationSubject,
+			Content:   "<div><pre><code>" + string(statusYaml) + "</code></pre></div>", // wrap with div/pre/code tags so the formatting remains intact in the delivered mail
```

Comment:
We don't need to send the status body. Just a message stating that the status is back to Ready is enough.